src/views/Timesheets/index.jsx

```diff
@@ -563,4 +582,38 @@ const Timesheets = (props) => {
   );
 };
 
-export default Timesheets;
+const Notice = () => (
+  <Container>
+    <Card>
+      <CardHeader title="Timesheets have moved" />
+      <CardContent>
+        <Typography>
+          Student Timesheets are now located on{' '}
+          <a
+            class="gc360_text_link"
+            href="https://gordon.criterionhcm.com/ui/#"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Criterion
+          </a>
+          . If you are attempting to enter time for a job worked prior to 8/20/23 please email{' '}
+          <a
+            class="gc360_text_link"
+            href="mailto:payroll@gordon.edu"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Payroll@Gordon.edu
+          </a>
+          .
+        </Typography>
+      </CardContent>
+    </Card>
+  </Container>
+);
+
+const switchOverDate = new Date('2023-08-22 12:00');
+const Component = () => (Date.now() > switchOverDate ? <Notice /> : <Timesheets />);
```

Comment:
Yes, that is true. I originally put this in to have it switch over on its own without us having to update the code over the weekend. The switch over happened earlier than I expected though, so I figured the simplest solution was to only update the one date line rather than changing the whole structure. Also, if we leave the Timesheets component extant but unused we get warnings that cause the linter to fail.

Reply:
Cool NBD